### DISCUSSION TITLE
February Release -  Support for Monorepos, customized commit messages and more

### DISCUSSION THREAD

**Discussion by aikoii** [GitbookIO/community#110] [Announcements]
Hey everyone! 

In case you missed it, we recently added support for Monorepos, and you can now customize your commit messages with Git Sync.

We also improved the search experience :mag_right: and tweaked change requests sidepanel to allow you to quickly follow up on all relevant changes. 

<img width="801" alt="CleanShot 2023-02-17 at 15 22 09@2x" src="https://user-images.githubusercontent.com/106068994/219652552-0d6cfd89-461b-4191-a34f-aae46244c86e.png">

Head on over to our [changelog](https://changelog.gitbook.com/february-2023/07-february-23-monorepos-customized-commit-messages-and-more) to read more! 👋

**Comment by rachellawson** [comment]
These announcements are awesome - thank you. What would be even more awesome is to alert the owners of orgs that the change is coming a day beforehand — just so we are not surprised by questions from the users.

**Reply by aikoii** [reply]
Thanks so much for your feedback, @rachellawson!

Right now our support team is small, but this is a good idea — we’ll look into ways to improve our updates and communications to give org owners more information 🙂.
